test/test.js

```diff
@@ -70,6 +70,10 @@ function limitedTest(name, testFn) {
   return test(name, async t => limiter(async () => testFn(t)));
 }
 
+function stripText(origText) {
+  return strip(origText.split('\r').join('').slice(0, -1));
```

Comment:
why split and splice and not a simple trimming `replace(/\r+/g, '')`?

Reply:
Why use regex when not needed?

Reply:
because replace is more readable than split & join?
(it's also faster, but this is not a real consideration here)